stryker/angular/README.md

```diff
@@ -0,0 +1,41 @@
+# Angular
+Stryker supports Angular projects using the Angular CLI starting from @angular/cli v6.1.0. Are you using an older version? Then there are some tips later in this document.
+
+## @angular/cli 6.1.0 and higher
```

Comment:
You have to mention minimum valid version of `@angular-devkit/build-angular` cause `~0.6.8` was causing stryker to hang in the middle of tests.

Reply:
I have documented the dependencies 👍